Tasks/DockerV2/task.loc.json

```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 2,
     "Minor": 150,
-    "Patch": 4
+    "Patch": 5
```

Comment:
why are we not changing for task.json of dockerv2?

Reply:
Task.json for DockerV2 is currently at patch 5, whereas for task.loc.json it was set to 4. I have now updated both to 6